featureflags/api.py

```diff
@@ -1,7 +1,7 @@
 from tenacity import retry_if_result, wait_exponential, \
```

Comment:
`api.py` is a wrapper around the OAPI generated client, **not** the generated code.